flutter/Graminsta/lib/auth.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/material.dart';
+import 'package:Graminsta/login.dart';
+
+class AuthPage extends StatefulWidget {
+  AuthPage({Key key}) : super(key: key);
+
+  @override
+  _AuthPageState createState() => new _AuthPageState();
+}
+
+class _AuthPageState extends State<AuthPage> with TickerProviderStateMixin {
+  PageController _pageController;
+  PageView _pageView;
+  int _currentPage = 0;
+
+  @override
+  void initState() {
+    super.initState();
+    _pageController = new PageController();
+    _pageView = new PageView(
+      controller: _pageController,
+      children: <Widget>[
+        new LoginPage(),
+        Text("Register"),
+      ],
+      onPageChanged: (index) {
+        setState(() {
+          _currentPage = index;
+        });
+      },
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+        body: new SafeArea(
+      child: new SingleChildScrollView(
+          child: new Container(
+              height: MediaQuery.of(context).size.height,
+              width: MediaQuery.of(context).size.width,
+              child: new Column(
+                mainAxisSize: MainAxisSize.max,
+                children: <Widget>[
+                  new SizedBox(
+                    height: 75,
```

Comment:
For constants, let's create a `spec` directory in the project root and put constants in it.

```
Graminsta/
Graminsta/spec/lib/spacing.dart
Graminsta/spec/lib/sizing.dart   // For sizes like font sizes, box heights
```

For example:

```
// spacing.dart
...
const mediumVerticalElementSpacing = 16;
const largexxx = 24;
const xLargexxx = 48;
```

Reply:
I create a directory under the lib dir if it's ok.
```
Graminsta/lib/spec/spacing.dart
Graminsta/lib/spec/sizing.dart
```
Sorry that I couldn't figure out how to import the package if the directory is under the root dir.

Reply:
try `import 'package:Graminsta/spec/spacing.dart';`

Reply:
The import statement works fine now. But it won't work if I move the file into the root directory.